support/templates/notulen-prepublish.hbs

```diff
@@ -25,6 +25,27 @@
         {{location}}
       </span>
     </p>
+    {{#if intermissions}}
+      <h2>Onderbrekingen</h2>
+      {{#each intermissions as |intermission|}}
+        <p typeof="http://mu.semte.ch/vocabularies/ext/Intermission" resource="{{intermission.uri}}">
+          <strong>
+            Van 
+            <span property='prov:startedAtTime' datatype="xsd:dateTime" content="{{intermission.startedAt.value}}">
```

Comment:
small remark: probably best to be consistent and use double quotes for all html attributes